service/eveapi.py

```diff
@@ -346,19 +346,6 @@ def __getattr__(self, this):
         # perform arcane attribute majick trick
         return _Context(self._root, self._path + "/" + this, self.parameters)
 
-    def __call__(self, **kw):
-        if kw:
-            # specified keywords override contextual ones
-            for k, v in self.parameters.iteritems():
-                if k not in kw:
-                    kw[k] = v
-        else:
-            # no keywords provided, just update with contextual ones.
-            kw.update(self.parameters)
-
-        # now let the root context handle it further
-        return self._root(self._path, **kw)
-
```

Comment:
This is removing code from a mostly third party module. Have we tested to make sure all API functions still function correctly? I'm not sure why we're trying to remove this bit of code